test/src/utils.rs

```diff
@@ -16,7 +16,7 @@ use std::thread::sleep;
 use std::time::{Duration, Instant};
 use tempfile::tempdir;
 
-pub const MEDIAN_TIME_BLOCK_COUNT: u64 = 11;
+pub const MEDIAN_TIME_BLOCK_COUNT: u64 = 37;
```

Comment:
It's weird, there are few const variables that are defined more than once, and they have to be equal. Why?